### DISCUSSION TITLE
MSSQL and Azure AD Authentication

### DISCUSSION THREAD

**Discussion by jbaarssen** [kedacore/keda#3773] [Q&A / Need Help]
Hi, 

In our current project we want to use the KEDA scaler for scaling Azure Durable Functions. The only thing is that our SQL database only allow Azure AD authentication. Is it possible for KEDA to use Managed Identity to connect with Azure SQL? I couldn't find anything related on this on the documentation.

Thanks

**Comment by JorTurFer** [comment]
[It seems that could be done](https://github.com/denisenkom/go-mssqldb#azure-active-directory-authentication) but atm KEDA doesn't support that. Are you willing to contribute it?
